src/components/full-view-accommodation/full-view-accommodation.tsx

```diff
@@ -21,40 +21,69 @@ export interface ShowPhotosProps {
   id: string;
   open: boolean;
   onClose: () => void;
+  handleOpen: () => void;
+  isMobile?: boolean;
 }
 
-export default function ShowPhotos({ id, open, onClose }: ShowPhotosProps) {
-  const { data } = UseGetAllMediaQuery(id);
+export default function ShowPhotos({ id, open, onClose, handleOpen, isMobile }: ShowPhotosProps) {
+  const [shouldFetch, setShouldFetch] = React.useState(false);
```

Comment:
As I can see, the main approach of using hooks on the project is the following

```ts
import React, { useCallback } from 'react'
...
const [state, setState] = useState(null)
```

Please, follow this one and get rid of `React.`